### DISCUSSION TITLE
Problem with Update-BTNotification

### DISCUSSION THREAD

**Discussion by Silloky** [Windos/BurntToast#212] [Q&A]
I'm trying to make a small PowerShell program  to display a toast notification that will show me in real-time how a copy job is doing. For the moment I haven't coded origin/copy size-compare program, so it's supposed to go up of 1% every second.

```powershell
$image = New-BTImage -Source "D:\Pictures\Profile image.png" -Crop Circle
$binding = @{
    text1 = "Downloading your files..."
    text2 = "Your files are being downloaded so they can be available offline."
    image = $image
    value = 0
}
$progress = New-BTProgressBar -Status "Downloading..." -Value $binding['value'] -Title "Download" -Verbose
New-BurntToastNotification -Text $binding['text1'], $binding['text2'] -AppLogo $binding['image'] -ProgressBar $progress -UniqueIdentifier "toast" -Verbose
foreach ($n in 1..100){
    $binding['value'] = $binding['value'] + 0.01
    $binding['text1'] = "hello"
    Write-Output $binding
    Update-BTNotification -UniqueIdentifier "toast" -DataBinding $binding -Verbose
    Start-Sleep 1
}
```

But it doesn't seem to update ! The `-Verbose` doesn't show any error, and `Write-Output $binding` is clearly evidence `$bindings` is being updated every second...

I don't know what I'm doing wrong...

Thanks for any answer,
Silloky

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
